client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -63,6 +63,13 @@ export class List extends React.Component {
 		notice: null,
 	};
 
+	componentDidUpdate( prevProps ) {
+
+		if ( this.props.selectedSite.ID !== prevProps.selectedSite.ID ) {
```

Comment:
I'd prolly use:
```
get( this.props, 'selectedSite.ID', null ) !== get( prevProps, 'selectedSite.ID', null )
```

To make sure we don't end up with undefined.

@klimeryk what do you think?

Reply:
That's a great tip! 🙌🏽 🙇‍♂️